interactions/client/auto_shard_client.py

```diff
@@ -68,6 +69,16 @@ def latencies(self) -> dict[int, float]:
         """
         return {state.shard_id: state.latency for state in self._connection_states}
 
+    @property
+    def start_time(self) -> datetime:
+        """The start time of the first shard of the bot."""
+        return next((state.start_time for state in self._connection_states), MISSING)  # type: ignore
+
+    @property
+    def start_times(self) -> list[datetime]:
```

Comment:
Nitpick. This would be more useful as a dict of {shard_id: start_time}

Reply:
Done 👍